app/models/grda_warehouse/health_emergency/isolation.rb

```diff
@@ -6,6 +6,13 @@
 
 module GrdaWarehouse::HealthEmergency
   class Isolation < GrdaWarehouseBase
-  include HealthEmergency
+    include HealthEmergency
```

Comment:
I don't think these are needed -- the prefix comes in from the module decl...

Reply:
These bring in the acts_as_paranoid and relations, no the table prefix.